integration_test/testgenerated/binary/api/servers.conjure.go

```diff
@@ -18,9 +18,10 @@ import (
 type TestService interface {
 	BinaryAlias(ctx context.Context, bodyArg io.ReadCloser) (io.ReadCloser, error)
 	BinaryAliasOptional(ctx context.Context) (*io.ReadCloser, error)
-	BinaryAliasAlias(ctx context.Context, bodyArg io.ReadCloser) (*io.ReadCloser, error)
+	BinaryAliasAlias(ctx context.Context, bodyArg *io.ReadCloser) (*io.ReadCloser, error)
```

Comment:
Making the argument a pointer here is a breaking change to code which implements a server resource with optional argument, but I think it is more correct and forces users to contemplate what to do about a nil value.

Reply:
Agree that this is more correct and forces users to handle the optionality. Similar to other recent changes, we need to be cautious about how this impacts current users. I imagine there are a lot of conjure defs that use optional parameters where it will need to be updated. We should get some more eyes here to raise awareness and provide guidance on the path forward.